courses/serializers.py

```diff
@@ -195,6 +209,38 @@ def get_description(self, instance):
         """Description"""
         return instance.page.description if instance.page else None
 
+    def get_start_date(self, instance):
+        """Start date"""
+        return (
+            models.CourseRun.objects.filter(course__program=instance, live=True)
+            .order_by("start_date")
+            .values_list("start_date", flat=True)
+            .first()
+        )
+
+    def get_end_date(self, instance):
+        """End date"""
+        return (
+            models.CourseRun.objects.filter(course__program=instance, live=True)
+            .order_by("end_date")
+            .values_list("end_date", flat=True)
+            .last()
+        )
+
+    def get_enrollment_start(self, instance):
```

Comment:
question: not dealing with `enrollment_end` yet now ?

Reply:
No, it was not asked for in the issue and @mbertrand said he doesn't need that information